packages/cli/src/WorkflowExecuteAdditionalData.ts

```diff
@@ -410,7 +410,14 @@ function hookFunctionsSave(parentProcessMode?: string): IWorkflowExecuteHooks {
 					workflowId: this.workflowData.id,
 				});
 
-				await restoreBinaryDataId(fullRunData, this.executionId, this.mode);
+				try {
+					await restoreBinaryDataId(fullRunData, this.executionId, this.mode);
+				} catch (e) {
+					logger.debug('Failed to restore binary data ID', {
```

Comment:
Would be nice to handle only the specific error here [instead of everything](https://testing.googleblog.com/2023/12/exceptional-exception-handling.html).

Also, since this isn't really something that is expected, wouldn't we want to log this either as an `error` or `warning`?

Reply:
Good idea, Tomi - changed the level to `errror`

Reply:
Changed - looks good? Would you mind approving?